docs/api/monitoring/monitoring-adapter.md

```diff
@@ -6,6 +6,15 @@ description: This document describes the interface that a monitoring adapter sho
 
 ## Monitoring Adapter API
 
+:::info TypeScript Support
+
+A monitoring adapter object has an associated TypeScript type:
+
+- Type Name - `MonitoringAdapter`
+- Package - `@resolve-js/core`
+
+:::
+
 A monitoring adapter implementation should expose the following interface:
```

Comment:
The table below has multiple list items. In this case, should "interface" be "interfaces"? Just checking.

Reply:
In this case, it is intended to be in singular form. By "interface" I mean "a collection of members that an object must expose so that it can be used as an adapter"

Reply:
Okay, thanks for the clarification.